### DISCUSSION TITLE
Sending requests to itself instead of sentry?

### DISCUSSION THREAD

**Discussion by kkmuffme** [stayallive/wp-sentry#114] [Q&A] [answered]
We see some `POST /api/5189343/store/` requests in our own access log, with requester user agent `sentry.php.wp-sentry-integration/4.6.0`

Sentry works for us (= we see the errors in sentry.io), so we were wondering what/why is happening here?

Are you using wp_remote_post under the hood or something? (and somehow this gets overwritten by a filter on our end)

**Comment by stayallive** [comment]
Ummm no... that is really interesting though! Are you also seeing the IP as being your own server?

We use the PHP SDK and their transports, WordPress is not involved in making the request to Sentry server so unless you have the DSN somehow set to your own server or have a filter messing with the SDK options (and mainly the DSN) I would not even know where to begin looking 😅

**Comment by kkmuffme** [comment]
Yes, the IP is our own server

**Comment by stayallive** [comment, marked as answer]
Hi @kkmuffme did you end up figuring this one out or solving it?

**Reply by kkmuffme** [reply]
Found the problem now (by accident). It was a misconfiguration error, caused by a bug in PHP's filter_input on CLI requests, where it fell back to sentry.io instead of aborting altogether.
Not an issue with WP sentry.

**Reply by stayallive** [reply]
Super interesting! Thanks for the update!
